core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -295,4 +295,12 @@ package object config {
         "above this threshold. This is to avoid a giant request takes too much memory.")
       .bytesConf(ByteUnit.BYTE)
       .createWithDefaultString("200m")
+
+  private[spark] val TASK_METRICS_TRACK_UPDATED_BLOCK_STATUSES =
+    ConfigBuilder("spark.taskMetrics.trackUpdatedBlockStatuses")
```

Comment:
Not sure if we want to document this somewhere?  If so what do you suggest there isn't any other config quite like this right now. Could either put in configuration doc or monitoring doc.